client/features/media/MediaDetail.jsx

```diff
@@ -113,14 +189,14 @@ class MediaDetail extends React.Component {
 
     if (this.state.watchWithUnsubscribed.length === 0) {
       unsubLengthStatement = 'Not available to watch on any other providers';
-    } else if (this.state.watchWithUnubscribed.length === 1) {
+    } else if (this.state.watchWithUnsubscribed.length === 1) {
       unsubLengthStatement = 'Also available on this other provider:';
     } else {
       unsubLengthStatement = 'Also available on these other providers:';
     }
 
     return (
-      <div id="MovieDetail">
+      <div id="mediaDetail">
```

Comment:
You can name this top-level component the same way as your React component ("MediaDetail").The style guide has been updated accordingly.

Reply:
that's awesome! Thank you!